### PR TITLE
Add Ubuntu 24.04 alias to PS

### PR DESCRIPTION
**Pull request by Sandido**
## Description
This PR adds a new alias for the new Ubuntu 24.04 image alias that was published recently. It also fixes broken aliases that were deprecated in the past 6 months and updates them to the current Azure Portal defaults.  

## Mandatory Checklist

- Please choose the target release of Azure PowerShell. (⚠️**Target release** is a different concept from **API readiness**. Please click below links for details.)
  - [x] [General release](../blob/main/CONTRIBUTING.md#target-release-types)
  - [ ] [Public preview](../blob/main/CONTRIBUTING.md#target-release-types)
  - [ ] [Private preview](../blob/main/CONTRIBUTING.md#target-release-types)
  - [ ] [Engineering build](../blob/main/CONTRIBUTING.md#target-release-types)
  - [ ] No need for a release

- [ ] Check this box to confirm: **I have read the [_Submitting Changes_](../blob/main/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/main/CONTRIBUTING.md) and reviewed the following information:**

* **SHOULD** update `ChangeLog.md` file(s) appropriately
    * For SDK-based development mode, update `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`.
        * A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header in the past tense. 
    * For autorest-based development mode, include the changelog in the PR description.
    * Should **not** change `ChangeLog.md` if no new release is required, such as fixing test case only.
* **SHOULD** regenerate markdown help files if there is cmdlet API change. [Instruction](../blob/main/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)
* **SHOULD** have proper test coverage for changes in pull request.
* **SHOULD NOT** adjust version of module manually in pull request

### CODE REVIEW

**Review by grizzlytheodore — APPROVED** [review]
this look good to me, as it's a simple json file change. Looks like you also added an adequate test. 

Let's resolve the CI check

**Review by grizzlytheodore — COMMENTED** [review]
Just seeing the CI failures. Let's resolve them

File: src/Compute/Compute.Test/ScenarioTests/ImageTests.ps1 [hunk 1]
```diff
@@ -342,123 +342,183 @@ function Test-DefaultImagesExistManual
         # UbuntuLTS test
         
         # Ubuntu2204 test
+        $vmname = "v1" + $rgname;
         $publisher = $images.Linux.Ubuntu2204.publisher;
```
Comment (grizzlytheodore) [inline src/Compute/Compute.Test/ScenarioTests/ImageTests.ps1:346]: what is going on in this test?

File: src/Compute/Compute/Strategies/ComputeRp/Images.json [hunk 2]
```diff
@@ -1,48 +1,60 @@
...
-                               "version":  "latest"
-                           },
-                  "Ubuntu2204":  {
-                                    "publisher":  "Canonical",
-                                    "offer":  "0001-com-ubuntu-server-jammy",
-                                    "sku":  "22_04-lts-gen2",
-                                    "version":  "latest"
-                                },
-                  "FlatcarLinuxFreeGen2":  {
-                                    "publisher":  "kinvolk",
```
Comment (grizzlytheodore) [inline src/Compute/Compute/Strategies/ComputeRp/Images.json:40]: why are these being deleted?

File: src/Compute/Compute/Strategies/ComputeRp/Images.json [hunk 3]
```diff
@@ -1,48 +1,60 @@
...
-                                    "offer":  "flatcar-container-linux-free",
-                                    "sku":  "stable-gen2",
-                                    "version":  "latest"
-                                }
+  "Linux": {
+    "CentOS85Gen2": {
+      "publisher": "OpenLogic",
+      "offer": "CentOS",
+      "sku": "8_5-gen2",
+      "version": "latest"
```
Comment (grizzlytheodore) [inline src/Compute/Compute/Strategies/ComputeRp/Images.json:7]: why are images other than the new Ubuntu2402 being added?

**Comment by grizzlytheodore** [comment]
do I need to delete and add the same images as this PR in swagger repos https://github.com/Azure/azure-rest-api-specs/blob/main/arm-compute/quickstart-templates/aliases.json?
@Sandido

File: src/Compute/Compute/Strategies/ComputeRp/Images.json [hunk 4]
```diff
@@ -1,48 +1,60 @@
...
+      "sku": "8-lvm-gen2",
+      "version": "latest"
+    },
+    "SuseSles15SP5": {
+      "publisher": "SUSE",
+      "offer": "sles-15-sp5",
+      "sku": "gen2",
+      "version": "latest"
     },
+    "Ubuntu2204": {
```
Comment (grizzlytheodore) [inline src/Compute/Compute/Strategies/ComputeRp/Images.json:33]: could you point to me where/how in the test this image alias is referenced?

File: src/Compute/Compute/Strategies/ComputeRp/Images.json (hunk 2 above)
Comment (grizzlytheodore) [reply src/Compute/Compute/Strategies/ComputeRp/Images.json:40]: they are overdue deprecated images right

**Comment by grizzlytheodore** [comment]
looks like the test is passing. is there anything needed on this PR?

**Comment by grizzlytheodore** [comment]
could you redirect this PR to https://github.com/Azure/azure-powershell/tree/feature/cplat-imageAlias so i can add my changes?

File: src/Compute/Compute.Test/ScenarioTests/ImageTests.ps1 (hunk 1 above)
Comment (Sandido) [reply src/Compute/Compute.Test/ScenarioTests/ImageTests.ps1:346]: @grizzlytheodore 
This test 
1. Creates a unique VM name.
2. Grabs the image data from the json file in PS. 
3. Tries to Get that image to ensure it still exists. 
4. Uses that image to make a VM. 
5. If any errors occur, bad.

**Comment by Sandido** [comment]
> do I need to delete and add the same images as this PR in swagger repos https://github.com/Azure/azure-rest-api-specs/blob/main/arm-compute/quickstart-templates/aliases.json? @Sandido

Correct!

**Pull request closed without merging**